activities.json

```
@@ -520,15 +520,15 @@
   },
   {
     "ciuName": null,
-    "description": "The ES Import Attributes and JSON modules proposal adds an inline syntax for module import statements to pass on more information alongside the module specifier, and an initial application for such attributes in supporting JSON modules in a common way across JavaScript environments.",
-    "id": "import-attributes",
+    "description": "The Import Conditions and JSON modules proposal adds an inline syntax for module import statements to pass on more information alongside the module specifier, and an initial application for such attributes in supporting JSON modules in a common way across JavaScript environments.",
+    "id": "import-conditions",
```

```suggestion
    "id": "import-attributes",
```

I'd prefer to leave the ID as-is since people may have links to it.